bmc-log-collector/bmc-http-client.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"crypto/tls"
+	//"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
+	"os"
+	"time"
+)
+
+// タイムアウトが必要？
+// Get from Redfish on iDRAC webserver
+func bmcClient(url string) ([]byte, error) {
```

Comment:
Change the function name appropriately.
`bmcClient` looks like a function that creates a client, but is actually making a request.